### PR TITLE
Add Enable/Disable all option for classifications

### LINKED ISSUE #2931: Allow disable/enable all for toggling classifications
**What existing feature could be improved?**
For layers with a lot of classifications e.g. Land Cover Type it's hard/cumbersome to turn on and off a lot of classifications in one go.

**Describe the improvement you'd like**
Provide a "Disable all" or "Enable all" option for toggling the classifications. 

**Additional context**

### PR DESCRIPTION
**Pull request by Benjaki2**
## Description

Fixes #2931

Add Enable/Disable classifications option.

Testing:

1. [Open page with land cover type layer active](http://localhost:3000/?t=2018-10-24-T17%3A04%3A50Z&l=MODIS_Combined_L3_IGBP_Land_Cover_Type_Annual,Reference_Labels(hidden),Reference_Features(hidden),Coastlines,VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor)
2. Go to settings for land cover type

## Further comments

If this is a relatively large or complex change, start a discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

## Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

@nasa-gibs/worldview

### CODE REVIEW

**Review by minniewong — APPROVED** [review]
Looks great!

File: web/js/components/layer/settings/settings.js [hunk 1]
```diff
@@ -320,6 +337,12 @@ const mapDispatchToProps = (dispatch) => ({
       setToggledClassification(layerId, classIndex, index, groupName),
     );
   },
+  toggleAllClassification: (layerId, disabledArray, index, groupName) => {
+    console.log(disabledArray);
```
Comment (jasontk19) [inline web/js/components/layer/settings/settings.js:341]: 🚨

**Review by jasontk19 — COMMENTED** [review]
Looks good!

Only a minor optional nitpick... it might be more intuitive to change the label to "enable/disable all" since it's in the same row as the toggle which does that?  Or just move the new toggle to the next line, above all the individual toggles with a label that says "All"?  

It's probably fine as is though, I doubt anyone will not understand what the new toggle switch is for.

Screen Shot 2020-10-27 at 3 25 22 PM

**Comment by Benjaki2** [comment]
@jasontk19 how does this look?
Screen Shot 2020-10-27 at 4 54 30 PM

File: web/js/components/layer/settings/settings.js (hunk 1 above)
Comment (Benjaki2) [reply web/js/components/layer/settings/settings.js]: whoops

**Comment by jasontk19** [comment]
> @jasontk19 how does this look?
> Screen Shot 2020-10-27 at 4 54 30 PM

Looks good!

**Review by jasontk19 — APPROVED** [review]

**Comment by jasontk19** [comment]
Also, looks like at least one of the failing tests may actually be related to these changes: https://travis-ci.org/github/nasa-gibs/worldview/jobs/739395171#L3140

**Pull request merged**